contracts/DCAFeeManager/DCAFeeManager.sol

```diff
@@ -218,19 +197,9 @@ contract DCAFeeManager is SwapAdapter, AccessControl, Multicall, IDCAFeeManager
   }
 
   /// @dev This version does not check the swapper registry at all
-  function _maxApproveSpenderIfNeeded(
-    IERC20 _token,
-    address _spender,
-    uint256 _minAllowance
-  ) internal {
+  function _maxApproveSpender(IERC20 _token, address _spender) internal {
     if (_spender != address(0)) {
```

Comment:
Just realized that this check doesn't make sense. We simply won't send the allowance target as part of the array. Let's remove it